litecommands-fabric/src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,18 @@
+{
+    "schemaVersion": 1,
+    "id": "litecommands",
+    "version": "3.3.4",
+    "name": "LiteCommands",
+    "description": "A simple command framework for Fabric.",
+    "authors": [
+        "Rollczi","huanmeng_qwq"
+    ],
+    "contact": {
+        "homepage": "https://github.com/Rollczi/LiteCommands",
+        "issues": "https://github.com/Rollczi/LiteCommands/issues"
+    },
+    "license": "MIT",
```

Comment:
this is for mods? I don't know if this should be in the library

Reply:
If the file is not present, it will cause the build to fail